image/roxctl/Dockerfile.rhtap

```diff
@@ -0,0 +1,53 @@
+# TODO: can we follow tags?
+FROM registry.access.redhat.com/ubi8/go-toolset:1.19 as builder
+
+WORKDIR /go/src/github.com/stackrox/rox/app
+
+# TODO: do we want to avoid this by COPYing each required directory separately?
+COPY . .
+
+RUN git config --global --add safe.directory /go/src/github.com/stackrox/rox/app && \
+    # We could add tag fetching to: https://github.com/redhat-appstudio/build-definitions/blob/main/task/git-clone/0.1/git-clone.yaml
+    # TODO: if git fetch fails, check if privileges/file owners are OK, usually only on local build because repo might have moved further.
+    # TODO: this prevents hermetic builds
+    git fetch --tags --force && \
+    mkdir -p image/bin
+
+ENV CI=1 GOFLAGS="" GOTAGS="release"
+
+RUN RACE=0 CGO_ENABLED=1 GOOS=linux GOARCH=$(go env GOARCH) BUILD_TAG=$(make tag) scripts/go-build.sh ./roxctl && \
+    cp bin/linux_$(go env GOARCH)/roxctl image/bin/roxctl
+
+# TODO: can we follow tags?
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest as app
+
+COPY --from=builder /go/src/github.com/stackrox/rox/app/image/bin/roxctl /usr/bin/roxctl
+
+# TODO: this prevents hermetic builds
+RUN microdnf upgrade -y --nobest && \
+    microdnf clean all && \
+    rpm --verbose -e --nodeps $(rpm -qa curl '*rpm*' '*dnf*' '*libsolv*' '*hawkey*' 'yum*') && \
+    rm -rf /var/cache/dnf /var/cache/yum
+
+LABEL \
+    com.redhat.component="rhacs-roxctl-container" \
+    name="rhacs-roxctl-rhel8" \
+    maintainer="Red Hat, Inc." \
+    # These labels are added to override the base image values.
+    description="The CLI for ACS" \
+    io.k8s.description="The CLI for ACS" \
+    io.k8s.display-name="roxctl" \
+    io.openshift.tags="rhacs,roxctl" \
+    summary="The CLI for ACS"
+
+    # TODO: what are these for & if they're required, how can we ?
+    # version="${CI_VERSION}" \
+    # "git-commit:stackrox/stackrox"="${CI_STACKROX_UPSTREAM_COMMIT}" \
+    # "git-branch:stackrox/stackrox"="${CI_STACKROX_UPSTREAM_BRANCH}" \
+    # "git-tag:stackrox/stackrox"="${CI_STACKROX_UPSTREAM_TAG}"
```

Comment:
I added these to downstream-built containers to be able to more directly tell from which upstream commit, branch and tag the code was built. It is possible to find out without the same labels but takes 6+ steps.

There's no code that depends on presence or absence of these labels but I suggest to still populate them as they seem to be handy.

Reply:
Yes, let's drop these since we're not provided any further build args anyways. (In https://github.com/redhat-appstudio/build-definitions/blob/main/task/buildah/0.1/buildah.yaml#L163-L169)

One thing that I'd optionally suggest to add is the name of the upstream repo. Something like:
```
	upstream-repo="github.com/stackrox/stackrox"
```

This is to drive folks who found problems with the container quicker to our project.